app/players/player.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+
+    nicknames = []

```

Comment:
WHat for you create that list? You never used it

Reply:
why do u need this variable, do u use it somewhere ?